components/tooltip/tooltip.js

```diff
@@ -855,7 +867,7 @@ class Tooltip extends RtlMixin(LitElement) {
 	_onTargetMouseLeave() {
 		clearTimeout(this._hoverTimeout);
 		this._isHovering = false;
-		this._updateShowing();
+		setTimeout(this._updateShowing.bind(this), 100); // delay to allow for mouseenter to fire if hovering on tooltip
```

Comment:
Nit: arrow functions allow you to avoid `bind`

```suggestion
		setTimeout(() => this._updateShowing(), 100); // delay to allow for mouseenter to fire if hovering on tooltip
```